src/electron/platform/android/setup/android-setup-deps.ts

```diff
@@ -11,4 +11,5 @@ export type AndroidSetupDeps = {
     hasExpectedServiceVersion: () => Promise<boolean>;
     installService: () => Promise<boolean>;
     hasExpectedPermissions: () => Promise<boolean>;
+    setTcpForwarding: () => Promise<boolean>;
```

Comment:
Returns a bool; is this a question? Should this function name begin with 'has'?

Reply:
I'm continuing the pattern of not throwing from the deps. A false value = an error. I'll move the catch handler up into the onEnter